steps/unable-to-enter/second-failed-attempt.tsx

```diff
@@ -0,0 +1,221 @@
+    firstFailedAttempt.loading || !firstFailedAttempt.result?.date
+      ? "Loading..."
+      : formatDate(new Date(firstFailedAttempt.result?.date), "d MMMM yyyy");
+
+  const reasonValue =
+    firstFailedAttempt.loading || !firstFailedAttempt.result?.value
+      ? "Loading..."
+      : firstFailedAttempt.result?.value;
+
+  const reasonLabel = failedAttemptReasonCheckboxes
+    .filter((checkbox) => reasonValue.includes(checkbox.value))
+    .map((checkbox) => checkbox.label);
+
+  return (
+    <PageAnnouncement title={"Previous attempt"}>
+      <List
+        items={[
+          <React.Fragment key="first">
+            First failed attempt: {dateString}
+          </React.Fragment>,
+          <React.Fragment key="second">
+            Reason: <List items={reasonLabel.map((label) => label)} />
+          </React.Fragment>,
+        ]}
+      />
+    </PageAnnouncement>
+  );
+};
+
+const step = {
+  title: PageTitles.SecondFailedAttempt,
+  heading: "Unable to enter the property",
+  step: {
+    slug: PageSlugs.SecondFailedAttempt,
+    nextSlug: PageSlugs.Pause,
+    submit: (nextSlug?: string): ReturnType<typeof makeSubmit> =>
+      makeSubmit([
+        {
+          slug: nextSlug as PageSlugs | undefined,
+          value: "Save and continue",
+          afterSubmit: storeSecondVisitDate,
+        },
+        {
+          cancel: true,
+          value: "Cancel",
+        },
+      ]),
+    componentWrappers: [
+      ComponentWrapper.wrapStatic(
+        new StaticComponent({
+          key: "one",
+          Component: PreviousFailedAttemptsAnnouncement,
+          props: {},
+        })
+      ),
+      ComponentWrapper.wrapStatic(
+        new StaticComponent({
+          key: "second-attempt-heading",
+          Component: Heading,
+          props: {
+            level: HeadingLevels.H2,
+            children: "Second attempt",
+          },
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "why-unable-to-enter",
+          Component: Checkboxes,
+          props: {
+            name: "why-unable-to-enter",
+            legend: (
+              <FieldsetLegend>
+                Why were you unable to enter the property? Did you observe
+                anything of concern?
+              </FieldsetLegend>
+            ) as React.ReactNode,
+            checkboxes: [
+              {
+                label: "Tenant not in",
+                value: "tenant not in",
+              },
+              {
+                label: "Not safe to enter property",
+                value: "not safe",
+              },
+              {
+                label: "Property may be abandoned",
+                value: "abandoned",
+              },
+              {
+                label: "There are signs of subletting",
+                value: "subletting",
+              },
+              {
+                label: "Other",
+                value: "other",
+              },
+            ],
```

Comment:
```suggestion
            checkboxes: failedAttemptReasonCheckboxes
```

And do the same for the first failed attempt.